tls/test/test_message.py

```diff
@@ -60,12 +61,14 @@ class TestCertificateRequestParsing(object):
 
     def test_parse_certificate_request(self):
         record = CertificateRequest.from_bytes(self.no_authorities_packet)
-        assert record.certificate_types == [ClientCertificateType.RSA_SIGN]
+        assert record.certificate_types == [
+            enums.ClientCertificateType.RSA_SIGN
+        ]
         assert len(record.supported_signature_algorithms) == 1
         assert record.supported_signature_algorithms[0].hash == \
-            HashAlgorithm.MD5
+            enums.HashAlgorithm.MD5
```

Comment:
I prefer parentheses here too, but I'm not sure what style guide this project is following.